docs/content/docs/recipes/shared-string.md

```diff
@@ -0,0 +1,454 @@
+---
+title: Using SharedString with React
+menuPosition: 4
+author: scottn12
+---
+
+In this tutorial, you'll learn how to use the `SharedString` distributed data structure (DDS) with [React](https://reactjs.org/). To learn more about `SharedString`, click [here](https://fluidframework.com/docs/data-structures/string/).
+
+
+To jump ahead into the finished demo, check out the [SharedString example in our FluidExamples repo](TODO_URL_HERE).
+
+The following image shows a textarea open in four browser. The same text is in all four. The cursor is in the bottom textarea in one browser.
```

Comment:
```suggestion
The following image shows a textarea open in four browsers. The same text is in all four. The cursor is in the bottom textarea in one browser.
```

Reply:
would "text area" instead of "textarea" be more appropriate here?  i would think the tutorial is more about a general text area rather than specifically a textarea element